services/consensuscontext/test/consensus_context_test.go

```diff
@@ -0,0 +1,70 @@
+package test
```

Comment:
Note that `RequestNewTransactionsBlock` and `RequestNewResultsBlock` are not implemented properly, you need to write component tests for them and make sure they are on par with spec

Reply:
They are old code I wrote with Tal in August. They are next in line for rewrite. This PR doesn't cover all of consensus context. Thinking of it, the PR name is misleading.